highfidelity/hifi
# Use setInterval() for cam position; Update model/registration; SPECTATOR button protections

**Test Plan:**
1. In HMD, open the Spectator Camera app.
2. Turn on the Spectator Camera.
3. Look at the camera model in front of you. Verify that the viewfinder preview appears in the "glass pane" of the camera model.
4. Point the lens of the camera at your face. Verify that your face appears centered in the viewfinder preview.
5. Ensure your framerate is 90. While holding the camera with the lens facing you, walk in any direction. Verify that the preview contains you and that the image is not "jittery".
6. Enable Developer Options and press Ctrl+R on your keyboard to reload all scripts. Verify that the camera disappears. Open your tablet and verify that only one "Spectator" button appears in the tablet.
